### ISSUE TITLE
Update Catalan translations

### ISSUE THREAD

**Issue by gsalvatella** [squidfunk/mkdocs-material#5008]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ca",
  "action.edit": "Edita aquesta pàgina",
  "action.skip": "Salta el contingut",
  "action.view": "Visualitza el codi font ⬅️",
  "announce.dismiss": "No ho tornis a mostrar ⬅️",
  "blog.archive": "Arxiva ⬅️",
  "blog.categories": "Categories ⬅️",
  "blog.categories.in": "a ⬅️",
  "blog.continue": "Continua llegint ⬅️",
  "blog.draft": "Esborrany ⬅️",
  "blog.index": "Torna a l'índex ⬅️",
  "blog.meta": "Metadades ⬅️",
  "blog.references": "Enllaços relacionats ⬅️",
  "clipboard.copy": "Còpia al porta-retalls",
  "clipboard.copied": "Copiat al porta-retalls",
  "consent.accept": "Accepta ⬅️",
  "consent.manage": "Gestiona la configuració ⬅️",
  "consent.reject": "Rebutja ⬅️",
  "footer": "Peu de pàgina ⬅️",
  "footer.next": "Següent",
  "footer.previous": "Anterior",
  "header": "Capçalera ⬅️",
  "meta.comments": "Comentaris",
  "meta.source": "Codi font",
  "nav": "Navegació ⬅️",
  "readtime.one": "1 min de lectura ⬅️",
  "readtime.other": "# min de lectura ⬅️",
  "rss.created": "Canal RSS ⬅️",
  "rss.updated": "Canal RSS de contingut actualitzat ⬅️",
  "search": "Cerca ⬅️",
  "search.placeholder": "Cerca",
  "search.share": "Comparteix ⬅️",
  "search.reset": "Neteja ⬅️",
  "search.result.initializer": "Inicialitzant cerca ⬅️",
  "search.result.placeholder": "Escriu per a començar a cercar",
  "search.result.none": "Cap document coincideix",
  "search.result.one": "1 document coincident",
  "search.result.other": "# documents coincidents",
  "search.result.more.one": "1 més en aquesta pàgina ⬅️",
  "search.result.more.other": "# més en aquesta pàgina ⬅️",
  "search.result.term.missing": "Desaparegut ⬅️",
  "select.language": "Selecciona la llengua ⬅️",
  "select.version": "Selecciona la versió ⬅️",
  "source": "Ves al repositori",
  "source.file.contributors": "Col·laboradors ⬅️",
  "source.file.date.created": "Creada",
  "source.file.date.updated": "Darrera actualització",
  "tabs": "Pestanyes ⬅️",
  "toc": "Taula de continguts",
  "top": "Torna a l'inici ⬅️"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by gsalvatella** [comment]
The icon for the catalan language in the [documentation](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) should look like that in the image. 
![cat-flag](https://user-images.githubusercontent.com/42438361/217380617-27513f17-9e47-498b-87af-ce4750ab36ef.png)

**Comment by squidfunk** [comment]
Thanks! Sadly, Twemoji, which we take the flags from, does not provide a Catalan flag, which is why we've chosen the closest option that is the Spanish flag (go to our [icon search](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/#search) and search for `flag`). We're aware that this is not perfect, but it's only meant for visual orientation. It's not political by any means. We'll add a note to the documentation.

**Comment by squidfunk** [comment]
Released as part of [9.0.12](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.12).
